### PR TITLE
Addition of SceneLayer Samples

### PR DESCRIPTION
**Pull request by macca400**
Added Scene Layer sample for WPF, UWP ,Droid ,iOS and Forms.

### CODE REVIEW

**Comment by macca400** [comment]
@ThadT can you review please?

File: src/WPF/ArcGISRuntime.WPF.Samples/Samples/Layers/SceneLayerUrl/SceneLayerUrl.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+using System;
+
+namespace ArcGISRuntime.WPF.Samples.SceneLayerUrl
+{
+    public partial class SceneLayerUrl
+    {
+        public SceneLayerUrl()
+        {
+            InitializeComponent();
+
```
Comment (ThadT) [inline src/WPF/ArcGISRuntime.WPF.Samples/Samples/Layers/SceneLayerUrl/SceneLayerUrl.xaml.cs]: This comment seems to get copied over to all new samples, but doesn't always apply (no UI is being created, e.g.).

File: src/WPF/ArcGISRuntime.WPF.Samples/Samples/Layers/SceneLayerUrl/SceneLayerUrl.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+            // Create the UI, setup the control references and execute initialization 
+            Initialize();
+        }
+
+        private void Initialize()
+        {
+            // Create new Scene
+            Scene myScene = new Scene();
+
+            // Set Scenes base map property
```
Comment (ThadT) [inline src/WPF/ArcGISRuntime.WPF.Samples/Samples/Layers/SceneLayerUrl/SceneLayerUrl.xaml.cs]: Scenes => Scene's

File: src/WPF/ArcGISRuntime.WPF.Samples/Samples/Layers/SceneLayerUrl/SceneLayerUrl.xaml.cs [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+            // Create a camera with cordinates showing layer data 
+            Camera camera = new Camera(48.378, -4.494, 200, 345, 65, 0);
+
+            // Assign the Scene to the SceneView
+            MySceneView.Scene = myScene;
+
+            // Set View point of Scene view using camera 
+            MySceneView.SetViewpointCameraAsync(camera);
+
```
Comment (ThadT) [inline src/WPF/ArcGISRuntime.WPF.Samples/Samples/Layers/SceneLayerUrl/SceneLayerUrl.xaml.cs]: Remove extra space (before and after end bracket for Initialize function).

File: src/WPF/ArcGISRuntime.WPF.Samples/Samples/Layers/SceneLayerUrl/SceneLayerUrl.xaml.cs [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+            var serviceUri = new Uri(
+               "https://scene.arcgis.com/arcgis/rest/services/Hosted/Buildings_Brest/SceneServer/0");
+
+            // Create new scene layer from the url
+            ArcGISSceneLayer sceneLayer = new ArcGISSceneLayer(serviceUri);
+
+            // Add created layer to the operational layers collection
+            myScene.OperationalLayers.Add(sceneLayer);
+
+            // Create a camera with cordinates showing layer data
```
Comment (ThadT) [inline src/WPF/ArcGISRuntime.WPF.Samples/Samples/Layers/SceneLayerUrl/SceneLayerUrl.xaml.cs]: cordinates => coordinates

File: src/UWP/ArcGISRuntime.UWP.Samples/Samples/Layers/SceneLayerUrl/SceneLayerUrl.xaml.cs [hunk 5]
```diff
@@ -0,0 +1,54 @@
+﻿// Copyright 2016 Esri.
```
Comment (ThadT) [inline src/UWP/ArcGISRuntime.UWP.Samples/Samples/Layers/SceneLayerUrl/SceneLayerUrl.xaml.cs]: 2017

File: src/Android/Xamarin.Android/Samples/Layers/SceneLayerUrl/SceneLayerUrl.cs [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+            // Add created layer to the operational layers collection
+            myScene.OperationalLayers.Add(sceneLayer);
+
+            // Create a camera with cordinates showing layer data 
+            Camera camera = new Camera(48.378, -4.494, 200, 345, 65, 0);
+
+            // Assign the Scene to the SceneView
+            _mySceneView.Scene = myScene;
+
+            // Set View point of Scene view using camera
```
Comment (ThadT) [inline src/Android/Xamarin.Android/Samples/Layers/SceneLayerUrl/SceneLayerUrl.cs]: "ViewPoint" and "SceneView" or "view point" and "scene view"

File: src/Forms/Shared/Samples/Layers/SceneLayerUrl/SceneLayerUrl.xaml.cs [hunk 7]
```diff
@@ -0,0 +1,56 @@
+﻿// Copyright 2016 Esri.
```
Comment (ThadT) [inline src/Forms/Shared/Samples/Layers/SceneLayerUrl/SceneLayerUrl.xaml.cs]: 2017

File: src/WPF/ArcGISRuntime.WPF.Samples/Samples/Layers/SceneLayerUrl/SceneLayerUrl.xaml.cs [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+            // Add created layer to the operational layers collection
+            myScene.OperationalLayers.Add(sceneLayer);
+
+            // Create a camera with cordinates showing layer data 
+            Camera camera = new Camera(48.378, -4.494, 200, 345, 65, 0);
+
+            // Assign the Scene to the SceneView
+            MySceneView.Scene = myScene;
+
+            // Set View point of Scene view using camera
```
Comment (ThadT) [inline src/WPF/ArcGISRuntime.WPF.Samples/Samples/Layers/SceneLayerUrl/SceneLayerUrl.xaml.cs]: "ViewPoint" and "SceneView" or "view point" and "scene view"

File: src/iOS/Xamarin.iOS/Samples/Layers/SceneLayerUrl/SceneLayerUrl.cs [hunk 9]
```diff
@@ -0,0 +1,83 @@
+// Copyright 2016 Esri.
```
Comment (ThadT) [inline src/iOS/Xamarin.iOS/Samples/Layers/SceneLayerUrl/SceneLayerUrl.cs]: 2017

File: src/iOS/Xamarin.iOS/Samples/Layers/SceneLayerUrl/SceneLayerUrl.cs [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+
+namespace ArcGISRuntimeXamarin.Samples.SceneLayerUrl
+{
+    [Register("SceneLayerUrl")]
+    public class SceneLayerUrl : UIViewController
+    {
+        // Constant holding offset where the SceneView control should start
+        private const int yPageOffset = 60;
+
+        // Create and hold reference to the used SceneView
```
Comment (ThadT) [inline src/iOS/Xamarin.iOS/Samples/Layers/SceneLayerUrl/SceneLayerUrl.cs]: Sounds weird ... how about just "Create a new SceneView control"?

**Comment by macca400** [comment]
@ThadT I've made changes in line with review comments. No VB samples as of yet I'm happy to add at a later date if users request

**Pull request merged**